rpm_spec/core-agent.spec.in

```diff
@@ -575,7 +583,7 @@ rm -f %{name}-%{version}
 %config(noreplace) /etc/qubes/rpc-config/qubes.InstallUpdatesGUI
 %config(noreplace) /etc/qubes/rpc-config/qubes.VMShell+WaitForSession
 %dir /etc/qubes/autostart
-%config(noreplace) /etc/default/grub.qubes
+%config(noreplace) /etc/grub.d/30_qubes
```

Comment:
`/etc/grub.d` is not the same as `/etc/default/grub.d`. And the latter is AFAIR Debian-only.

Reply:
I have restored it.